tests/test_slash_assets.py

```
@@ -0,0 +1,67 @@
+                self.assertTrue(
+                    self.base_dir.resolve() in prompt_path.parents
+                    or prompt_path == self.base_dir.resolve(),
+                    "Prompt file must reside within the slash directory",
+                )
+
+    def _load_commands(self) -> Dict[str, Dict[str, object]]:
+        with CONFIG_PATH.open(encoding="utf-8") as handle:
+            try:
+                data = json.load(handle, object_pairs_hook=_object_pairs_hook)
+            except (json.JSONDecodeError, DuplicateKeyError) as exc:
+                raise AssertionError(f"Failed to load slash config: {exc}") from exc
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)